fix(astro): dynamically import actions (#13510)
withastro/astro: .changeset/clever-tigers-dig.md

---
'astro': patch
---

> Fixes a case where `astro:env` secrets used in actions would run too early and cause errors

Author: ```suggestion
Fixes a case where `astro:env` secrets used in actions would not be available
```
